README.md

```diff
@@ -122,25 +126,21 @@ L:asm:x86_32 > sys open
 │ openat │ 0x127 │ int dfd              │ const char *filename │ int flags    │ umode_t mode │
 └────────┴───────┴──────────────────────┴──────────────────────┴──────────────┴──────────────┘
 ```
-This prints possible variants of syscall depending on entered pattern. Syscall table for searching depends on chosen architecture and OS (Operating System). In this case it's ```x86_32``` and ```Linux```.
+``sys`` prints a list of possible variants for the provided syscall. The syscall table that shellen searches depends on the chosen architecture and operating system (OS). In this case, the architecture is ```x86_32``` and the OS is ```Linux```.
 
 
-### Common shellcodes
-Shellen can show you a list of common shellcodes depending on your keyword. The example of using placed in ```Pictures``` section. It uses API of shell-storm.org site (thanks to the author!). You can use it like this:
```

Comment:
I removed the sentence on the "Pictures" section because I did not see a picture that showed the output of the keyword command.